File: DataGridView buttons (C#)/[C#]-DataGridView buttons (C#)/C#/SqlDataOperations/Customer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SqlDataOperations
{
    public class Customer
    {
        public int CustomerIdentifier { get; set; }
        public string CompanyName { get; set; }
        public string ContactName { get; set; }
        public string Address { get; set; }
        public string City { get; set; }
        public string PostalCode { get; set; }
        public string Country { get; set; }
        public int ContactTypeIdentifier { get; set; }
        public string ContactTitle { get; set; }
    }
}

File: Find duplicate records via SQL or LINQ in database table/[C#][VB]-Find duplicate records via SQL or LINQ in database table/C# and VB/DataOperations_VB/Customer.vb
Public Class Customer
    Public Property Identifier As Integer
    Public Property CompanyName As String
    Public Property ContactName As String
    Public Property ContactTitle As String
    Public Property Address As String
    Public Property City As String
    Public Property PostalCode As String
    Public Property Exists As Boolean

    Public Sub New()
    End Sub
    Public ReadOnly Property AlreadyExistsData As String
        Get
            Return String.Format("Already exists:'{0}','{1}','{2}','{3}','{4}','{5}'", CompanyName, ContactName, ContactTitle, Address, City, PostalCode)
        End Get
    End Property
    Public ReadOnly Property DoesNotExistsData As String
        Get
            Return String.Format("Okay to add:'{0}','{1}','{2}','{3}','{4}','{5}'", CompanyName, ContactName, ContactTitle, Address, City, PostalCode)
        End Get
    End Property
    ''' <summary>
    ''' Used for demoing if a record exists
    ''' </summary>
    ''' <returns></returns>
    ''' <remarks></remarks>
    Public Overrides Function ToString() As String
        Return String.Format("'{0}' '{1}' '{2}' '{3}' '{4}' '{5}'", CompanyName, ContactName, ContactTitle, Address, City, PostalCode)
    End Function
End Class
